src/utils/date.js

```diff
@@ -29,3 +29,9 @@ export const convertToDateFormat = (string) => {
   const dateAsString = `${string.date}T${string.time}`
   return convertDate(dateAsString)
 }
+
+export const sortedByDateAdded = (data) => {
+  data.forEach((job) => (job.dateAdded = convertDate(job.dateAdded)))
+
+  return data.sort((a, b) => b.dateAdded - a.dateAdded)
+}
```

Comment:
Looks like there's an opportunity for a mini-refactor here, as both this and the `sortedByDate` function above do similar work.

How about something like this to replace both? (Untested)

```
export const sortObjectsByDateKey = (objects, dateKey) => {
  objects.forEach((o) => { o[dateKey] = convertDate(o[dateKey]) })

  return objects.sort((a, b) => b[dateKey] - a[dateKey])
}
```

Which could be called like this:

```
sortObjectsByDateKey(jobs, 'dateAdded')

// OR

sortObjectsByDateKey(jobs, 'dateRaised')
```

Later this could be extended to accept a sorting direction.